ofdb-boundary/src/lib.rs

```diff
@@ -596,24 +596,24 @@ impl From<ClearanceForPlace> for e::clearance::ClearanceForPlace {
     }
 }
 
-#[derive(Serialize)]
+#[derive(Serialize, Deserialize)]
 #[cfg_attr(feature = "extra-derive", derive(Debug, Clone, PartialEq, Eq))]
 pub struct ResultCount {
     pub count: u64,
 }
 
 #[derive(Serialize, Deserialize)]
-#[cfg_attr(feature = "extra-derive", derive(Debug, PartialEq))]
-pub struct LatLonDegrees(f64, f64);
+#[cfg_attr(feature = "extra-derive", derive(Clone, Debug, PartialEq))]
+pub struct LatLonDegrees(pub f64, pub f64);
```

Comment:
I think it's better to use explicit access methods.
Here is a good example:

https://github.com/kartevonmorgen/openfairdb/blob/52b939d0f6cfb37f1ea1cc7015696fecb3e2ddce/ofdb-entities/src/geo.rs#L98